### PR TITLE
Parameter to allow user to force the sealing mechanism

### PR DESCRIPTION
**Pull request by gavofyork**

### CODE REVIEW

**Comment by gavofyork** [comment]
Fixes #917

File: miner/src/transaction_queue.rs [hunk 1]
```diff
@@ -348,7 +348,12 @@ impl TransactionQueue {
 		where T: Fn(&Address) -> AccountDetails {
 
 		txs.into_iter()
-			.map(|tx| self.add(tx, &fetch_account))
+			.map(|tx| {
+				trace!(target: "miner", "adding...");
```
Comment (NikVolf) [inline miner/src/transaction_queue.rs]: probably better moving to `add` itself and append transaction hash to logging

**Comment by NikVolf** [comment]
merged @debris  & @tomusdrw  pr here and removed em locally

**Comment by arkpar** [comment]
Code looks good, but I'm not sure this feature is actually useful anymore.

**Comment by gavofyork** [comment]
it will continue to be useful for when we're managing pending transactions' chain state, at least until https://github.com/ethcore/parity/issues/925 is addressed.

**Pull request merged**